change/@ni-nimble-components-a1f39dbd-cf7d-46e3-91a4-9b43c61de216.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Could you add a note to #309 mentioning this research and linking to this README?

Reply:
Done